pipeline/terraform/stack/service_image_id_minter.tf

```diff
@@ -0,0 +1,78 @@
+module "image_id_minter_queue" {
+  source          = "git::github.com/wellcomecollection/terraform-aws-sqs//queue?ref=v1.1.2"
+  queue_name      = "${local.namespace_hyphen}_image_id_minter"
+  topic_arns      = [module.merger_images_topic.arn]
+  aws_region      = var.aws_region
+  alarm_topic_arn = var.dlq_alarm_arn
+}
+
+module "image_id_minter" {
+  source          = "../modules/service"
+  service_name    = "${local.namespace_hyphen}_image_id_minter"
+  container_image = local.id_minter_image
+
+  security_group_ids = [
+    aws_security_group.service_egress.id,
+    aws_security_group.interservice.id,
+    var.rds_ids_access_security_group_id,
+  ]
+
+  cluster_name = aws_ecs_cluster.cluster.name
+  cluster_arn  = aws_ecs_cluster.cluster.id
+
+  namespace_id = aws_service_discovery_private_dns_namespace.namespace.id
+
+  env_vars = {
+    metrics_namespace    = "${local.namespace_hyphen}_image_id_minter"
+    messages_bucket_name = aws_s3_bucket.messages.id
+
+    queue_url       = module.image_id_minter_queue.url
+    topic_arn       = module.image_id_minter_topic.arn
+    max_connections = local.id_minter_task_max_connections
+
+    logstash_host = local.logstash_host
+  }
+
+  secret_env_vars = {
+    cluster_url = "catalogue/id_minter/rds_host"
+    db_port     = "catalogue/id_minter/rds_port"
+    db_username = "catalogue/id_minter/rds_user"
+    db_password = "catalogue/id_minter/rds_password"
```

Comment:
Do we want to create different users for different services?

Reply:
Interesting point, I don't _think_ so as they're identical in every way apart from their location